clearwater-sip-stress-coreonly.root/usr/share/clearwater/bin/run_stress

```diff
@@ -144,14 +145,27 @@ shuffle(callees)
 register_csv = NamedTemporaryFile(prefix="sipp_register")
 invite_csv = NamedTemporaryFile(prefix="sipp_calls")
 
+# For wildcard IMPU testing, we expect wildcarded IMPUs to prefix the
+# wildcarded part with a specific character. This makes it easy to mock out
+# HSSes and other components which recognise that these are wildcards.
+WILDCARD_DELIMITER = "$"
+
 register_csv.write("SEQUENTIAL\n")
 for s in registers:
-    register_csv.write("{}\n".format(s))
+    if args.wildcards:
+        register_csv.write("{}{}000\n".format(s, WILDCARD_DELIMITER))
```

Comment:
Ah, I think I understand this now (having seen the Husk changes) - worth a comment?